packages/node_modules/pouchdb-utils/src/guardedConsole.js

```diff
@@ -1,6 +1,6 @@
 function guardedConsole(method) {
   /* istanbul ignore else */
-  if (console !== 'undefined' && method in console) {
+  if (typeof console !== 'undefined' && typeof console[method] === 'function') {
```

Comment:
tiny nit, once conditions start getting longer in length I usually find it cleaner to split them, so

    if (typeof console !== 'undefined' && 
        typeof console[method] === 'function') {

Looks like this is exactly on the limit (80) and is fine, just a heads up for future

Reply:
👍 great to know the house style 🙂